frontend/src/App.jsx

```diff
@@ -50,7 +51,8 @@ function App() {
           <Switch>
             <Route exact path="/psychologue/login/:token?" component={Login} />
             {user && <Route path="/psychologue/" component={PsychologistRouter} />}
-            <Route exact path="/trouver-un-psychologue" component={FindPsychologist} />
+            <Route exact path="/psy/:psyId" component={PublicPsychologistProfile} />
```

Comment:
To avoid confusion between `psy` (public) and `psychologue` (private), we could have kept `/trouver-un-psychologue/:psyId` (not blocking)